docs/UserGuide.md

```diff
@@ -137,11 +137,22 @@ Format: `edit INDEX [n/NAME] [i/NRIC] [p/CONTACT NUMBER] [e/EMAIL] [a/ADDRESS] [
 * Edits the client policy details at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * **At least one** of the optional fields must be provided.
 * Existing values will be updated to the input values.
+* ❗️**Note:** 
```

Comment:
Can use the `<div markdown="block" class="alert alert-info">` to wrap around these!